src/stream/src/executor/over_window/general.rs

```diff
@@ -382,7 +386,16 @@ impl<S: StateStore> OverWindowExecutor<S> {
                                     yield chunk;
                                 }
                             }
-                            _ => panic!("other cases should not exist"),
+                            _ => {
+                                let vnode =
+                                    this.state_table.compute_vnode_by_pk(&key.pk);
+                                let raw_key = serialize_pk_with_vnode(
+                                    &key.pk,
+                                    this.state_table.pk_serde(),
+                                    vnode,
+                                );
+                                panic!("other cases should not exist. raw_key: {:?}, existed: {:?}, new: {:?}", raw_key, tp1, tp2);
```

Comment:
Is it expected ?

Reply:
I think it is necessary to output content of wrong key in panic